SharedComponents/AntVideo/Block/FolkLost.cs

```diff
@@ -1,32 +0,0 @@
-﻿using System.IO;
-
-namespace AntMe.SharedComponents.AntVideo.Block {
-    internal class FolkLost : Lost, ISerializable {
-        private int folkId;
-
-        public FolkLost(int folkId, int id) : base(id) {
-            this.folkId = (ushort) folkId;
-        }
-
-        public FolkLost(Stream stream) : base(stream) {
-            Deserialize(stream);
-        }
-
-        public int FolkId {
-            get { return folkId; }
-        }
-
-        #region ISerializable Member
-
-        public override void Serialize(Stream outputstream) {
-            base.Serialize(outputstream);
-            Serializer.SendUshort(outputstream, folkId);
-        }
-
-        public new void Deserialize(Stream stream) {
-            folkId = Serializer.ReadUShort(stream);
-        }
-
```

Comment:
das ist ne schlechte Idee. Das hier ist teil des AntVideo Protokolls. Mag sein, dass es nicht verwendet wird aktuell, aber das Protokoll erlaubt ein verschwindendes Volk mitten in der Simulation.

Reply:
Kommando zurück. Schau mir gerade das Protokoll an und sehe kein "FolkLost". Entweder es gibt einen Namensmissmatch zwischen Enum und Klassen (Folk vs. Colony vs. Team) oder das Ding war wirklich eine tote Altlast...?